dist/manifest.json

```diff
@@ -13,8 +13,8 @@
   "homepage_url": "http://alpheios.net",
   "short_name": "Alpheios",
   "permissions": [
-    "http://*/",
-    "https://*/",
+    "http://*/*",
```

Comment:
this is interesting. I can see why the trailing / might have been a problem but why not just remove it?

Reply:
That's because, according to FF rules, it should match a `/^(https?|wss?|file|ftp|\*):\/\/(\*|\*\.[^*/]+|[^*/]+)\/.*$/` pattern and, without a trailing slash, it won't. As a result, FF will print a warining message on the extensions page. 

At the same time, `http://*` triggers no issues in Chrome.

I don't know if FF has any reasons to set a pattern as the above, but I think we probably have no choice but to obey its requirements.

Reply:
ok thanks.